.changeset/kind-mugs-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': patch
```

Comment:
Should be minor since new APIs introduced.

Reply:
Changed